library/bigip_selfip.py

```diff
@@ -86,7 +86,19 @@
       validate_certs: "no"
       vlan: "vlan1"
   delegate_to: localhost
-
+- name: Create Self IP with a Route Domain
```

Comment:
I didn't see any spaces between the other examples, let me know if you still think there should be spaces between the examples...